javascriptv3/example_code/bedrock-runtime/tests/invoke-model.integration.test.js

```diff
@@ -0,0 +1,22 @@
+import { describe, it, expect } from "vitest";
```

Comment:
You might want to make a [vite.config.js](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javascriptv3/example_code/cloudwatch/vite.config.js) at the `bedrock-runtime` level.

Reply:
Great idea, done.